code/game/objects/items/weapons/clown_items.dm

```diff
@@ -143,13 +143,12 @@
 	item_state = "gold_horn"
 
 /obj/item/weapon/bikehorn/golden/attack()
-	..()
 	flip_mobs()
+	return ..()

```

Comment:
so the result of the parent call is returned, which helps keep the oo chain unbroken.

other options include . = ..()